ResidentsSocialCarePlatformApi.Tests/V1/UseCase/GetRelationshipsByPersonIdUseCaseTests.cs

```diff
@@ -25,12 +26,13 @@ public void SetUp()
         [Test]
         public void WhenThereAreNoPersonalRelationships_ReturnsEmptyLists()
         {
+            var request = new GetRelationshipsRequest() { PersonId = 123456789 };
             PersonalRelationships noMatchingRelationships = new PersonalRelationships();
-            _mockSocialCareGateway.Setup(x => x.GetPersonalRelationships(123)).Returns(noMatchingRelationships);
+            _mockSocialCareGateway.Setup(x => x.GetPersonalRelationships(It.IsAny<long>())).Returns(noMatchingRelationships);
 
-            var response = _classUnderTest.Execute(123);
+            var response = _classUnderTest.Execute(request);
 
-            response.PersonId.Equals(123);
+            response.PersonId.Equals(request.PersonId);
```

Comment:
Are we testing that these should be equivalent?

.Equals will return true if equivalent, otherwise false but our test won't fail if false is returned.

We might be looking for `response.PersonId.Should().Be(request.PersonId);`

Reply:
Ahhh I think this was as a result of some copy and pasta. Definitely prefer the FluentAssertation's way!